libs/SalesforceHybridSDK/HybridPluginTestAppTests/ForcetkTestSuite.m

```diff
@@ -35,10 +35,6 @@ - (void)setUp
 }
 
 
-- (void)testGetInfo {
```

Comment:
This test is in SDKInfoTestSuite.m
(bad copy/paste when creating ForcetkTestSuite.m)